samples/vxflex_v230_k8s_121.yaml

```diff
@@ -47,6 +47,28 @@ spec:
         - name: X_CSI_HEALTH_MONITOR_ENABLED
           value: "false"
 
+      #"controller.nodeSelector" defines what nodes would be selected for pods of controller deployment
+      # Leave as blank to use all nodes
+      # Allowed values: map of key-value pairs
+      # Default value: None
+      # Examples:
+      #  node-role.kubernetes.io/control-plane: ""
+      nodeSelector:
+      #  node-role.kubernetes.io/control-plane: ""
```

Comment:
Please add this for master as well for both controller and node. Applies to all samples except 1.24

Reply:
Oops, I just merged the PR before I saw this comment. @abhi16394  my apologies for that, could you open another PR for that.